### DISCUSSION TITLE
请问下群晖chat怎么对接青龙面板的通知

### DISCUSSION THREAD

**Discussion by testwhite** [whyour/qinglong#1897] [Q&A]
请问下群晖chat怎么对接青龙面板的通知

**Comment by v2cn** [comment]
1. 打开SynologyChat套件，点击Chat界面"右上角头像->整合->传入的webhook
2. 新建个自定义的名字和频道（譬如：通知），下面的通知地址是默认的不用动。
3. 将webhook url的地址复制下，准备填写到青龙面板的群辉chat通知参数配置上。
4. 重复上述方式，点击头像，点击整合，选择第二个选项 发出的webhook。
5. 点击创建按钮会，在最下面生成一个令牌，将这个令牌直接复制到青龙面板即可（创建界面复制令牌后，可以点取消退出）。
参考文章：https://www.pwadmin.net/article/NAS

**Reply by testwhite** [reply]
转载我的文章请标明转载出处，码字不易。https://www.playezu.com/731734.html
